corehq/apps/domain/templates/domain/activate_transfer_domain.html

```diff
@@ -12,7 +12,7 @@
       {% if transfer %}
         <p class="lead">
           {% blocktrans with domain=transfer.domain%}
-          By clicking "accept" below you acknowledge that you accept full ownership of this project space ("{{ domain }}").  You agree to be bound by the terms of Dimagi's <a href="https://www.commcarehq.org/eula/">End User License Agreement</a>.  By accepting this agreement, your are acknowledging you have permission and authority to accept Dimagi's End User License Agreement.
+          By clicking "accept" below You acknowledge that You accept full ownership of this project space ("{{ domain }}").  You agree to be bound by the terms of Dimagi's <a href="https://www.commcarehq.org/eula/">End User License Agreement</a>.  By accepting this agreement, your are acknowledging you have permission and authority to accept Dimagi's End User License Agreement. A Dimagi representative will notify you when the transfer is complete.
```

Comment:
yea, i really think those should stay lower case. the actual spec has the text and they're capitalized in that (as well as the extra sentence that was added). wasn't sure if it was some sort of legal jargon